@here/harp-mapview/lib/Tile.ts

```diff
@@ -1461,10 +1478,50 @@ export class Tile implements CachedResource {
                         false
                     );
 
+                    extrudedObjects.push({
+                        object: edgeObj,
+                        materialFeature: false
+                    });
+
                     this.registerTileObject(edgeObj);
                     objects.push(edgeObj);
                 }
 
+                // animate the extrusion of buildings
+                const animatedExtrusionHandler = this.mapView.animatedExtrusionHandler;
+                if (
+                    isExtrudedPolygonTechnique(technique) &&
+                    animatedExtrusionHandler !== undefined
+                ) {
+                    extrudedObjects.push({
+                        object,
+                        materialFeature: true
+                    });
+
+                    const extrudedPolygonTechnique = technique as ExtrudedPolygonTechnique;
```

Comment:
This alias is not needed. Typescript is smart enough to detect that the type of technique is ExtrudedPolygonTechnique b/c you checked it above